allowed-breaking-changes.txt

```diff
@@ -56,3 +56,7 @@ incompatible-argument:@aws-cdk/aws-ecs.TaskDefinition.addVolume
 # We made properties optional and it's really fine but our differ doesn't think so.
 weakened:@aws-cdk/cloud-assembly-schema.DockerImageSource
 weakened:@aws-cdk/cloud-assembly-schema.FileSource
+
+# We changed some APIs to use non-deprecated counterparts
+changed-type:@aws-cdk/core.ConstructNode.metadata
+incompatible-argument:@aws-cdk/core.Stack.reportMissingContext
```

Comment:
This is a breaking change on a stable part of the CDK. This is not ideal, but it's likely that the number of CDK users using this API is very low, and the cost of fixing this on the user's end is very simple.

Any other ideas on how we can do this?

Reply:
I don't think there is a way. I will agree with you on `reportMissingContext()`. I'm not sure about `node.metadata`...

Reply:
In general, any ACTUAL REAL breaking changes will have to be performed on `v2-main`...

Reply:
I'll come up with a different strategy. I may have some ideas.